doc/source/ansys_tools_path_api/path.rst

```diff
@@ -15,5 +15,4 @@ of the installation path of the Mechanical application.
    find_mechanical
    get_mechanical_path
    change_default_mechanical_path
-   save_mechanical_path
-
+   save_mechanical_path
```

Comment:
@koubaa Where should this file be referenced in the docs? api/index.rst?

Reply:
I suggest api/path.rst which is linked to by api/index.rst